src/python_testing/TC_WEBRTCR_2_1.py

```diff
@@ -146,7 +146,7 @@ async def test_TC_WebRTCR_2_1(self):
         self.step(3)
         # Prompt user with instructions
         prompt_msg = (
-            "\nPlease commission the server app from DUT:\n"
+            f"\nPlease commission the server app with {params} from DUT:\n"
             f"  pairing onnetwork 1 {passcode}\n"
```

Comment:
shall we also update the command? otherwise, it is not aligned with the prompt, or we expect the user to figure out the command for QR code?